i18n/en.json

```
@@ -5498,6 +5498,14 @@
     "id": "store.sql_bot.update.updating.app_error",
     "translation": "We encountered an error updating the bot"
   },
+  {
+    "id": "store.sql_channel.CountUnreadSince.app_error",
```

This should use snake_case for the method name like
```suggestion
    "id": "store.sql_channel.count_unreads_since.app_error",
```